schema/compose-spec.json

```diff
@@ -419,7 +419,8 @@
                     "properties": {
                       "propagation": {"type": "string"},
                       "create_host_path": {"type": ["boolean", "string"]},
-                      "selinux": {"type": "string", "enum": ["z", "Z"]}
+                      "recursive": {"type": "string", "enum": ["z", "Z"]},
+                      "selinux": {"type": "string", "enum": ["enabled", "disabled", "writable", "readonly"]}
```

Comment:
Inversion?
```suggestion
                      "selinux": {"type": "string", "enum": ["z", "Z"]},
                      "recursive": {"type": "string", "enum": ["enabled", "disabled", "writable", "readonly"]}
```

Reply:
indeed. I'm trying to reduce my caffeine addiction, but this seems to come with side effects